parser/parser.y

```diff
@@ -13774,9 +13774,10 @@ RevokeRoleStmt:
 
 /**************************************LoadDataStmt*****************************************
  * See https://dev.mysql.com/doc/refman/5.7/en/load-data.html
+ * for load stmt with format see https://github.com/pingcap/tidb/issues/40499
```

Comment:
It seems the issue doesn't update to the latest syntax yet.

Reply:
done